### PR TITLE
Fixes #108 

### LINKED ISSUE #108: Enable SourceLink
**Motivation:** By adding a few SourceLink properties to the `csproj` files, we could allow developers to debug their apps with a full source code of our SDKs (referenced from within the apps as NuGet packages). Even though the SDK is present in the app as a NuGet package, Visual Studio will silently download its source files for the purpose of debugging.

**Howto:** https://github.com/dotnet/sourcelink#using-sourcelink

**Example:** 
https://github.com/ctaggart/SharpYaml/blob/de09b74f0ea2a470d90eaab04b049d265e59fda0/SharpYaml/Directory.Build.props

Internal reference: DC-535

### PR DESCRIPTION
**Pull request by Simply007**
### Motivation

Fixes #108 

### Checklist
- [x] Code follows coding conventions held in this repo
- [x] Tests are passing
- [x] Docu has been updated (if applicable)
- [ ] Temporary settings (e.g. project ID used during development and testing) have been reverted to defaults

### How to test

1. After the documentation review will be done publish a beta on the nuget server.
2. Try to create an application referencing packages
3. Try to debug source code using source link by the readme instructions

If everything is OK. release major version of the package.

### CODE REVIEW

File: KenticoCloud.Delivery/KenticoCloud.Delivery.csproj [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;net45</TargetFrameworks>
-    <Version>5.0.0</Version>
```
Comment (petrsvihlik) [inline KenticoCloud.Delivery/KenticoCloud.Delivery.csproj]: can you please check that version patching in appveyor works fine if you remove this?
how to verify:
- check that assembly & fileversions are set correctly (after a package is built in appveyor)
- check that sdk tracking header is set properly for outcoming http requests

File: KenticoCloud.Delivery/KenticoCloud.Delivery.csproj [hunk 2]
```diff
@@ -14,6 +13,7 @@
     <RepositoryType>git</RepositoryType>
     <PackageTags>kentico;mvc;aspnet;aspnetmvc</PackageTags>
     <GenerateAssemblyInfo>true</GenerateAssemblyInfo>
+    <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
```
Comment (petrsvihlik) [inline KenticoCloud.Delivery/KenticoCloud.Delivery.csproj]: i'd also set [PublishRepositoryUrl ](https://github.com/dotnet/sourcelink#using-sourcelink) to true

**Comment by petrsvihlik** [comment]
@MartinaFarkasova  @juraju-kentico  could you guys review changes in the readme? thx!

File: KenticoCloud.Delivery/KenticoCloud.Delivery.csproj (hunk 1 above)
Comment (Simply007) [reply KenticoCloud.Delivery/KenticoCloud.Delivery.csproj]: I have tried to download packages (artifacts) built by AppVeyor for v. 6.0.0-beta9 
* https://ci.appveyor.com/project/kentico/deliver-net-sdk/build/6.0.0-beta9

Tried to take a look to the dll properties:
* image

I have tried to debug code uding **SourceLink** 😈 :
```
var client = new DeliveryClient("975bf280-fd91-488c-994c-2f04416e5ee3");
var request = client.GetItemAsync("on_roasts");
var result = request.Result;
```
And the version was correctly when sending the request:
* 
image

So I assume that Version tag is not necessary.

File: KenticoCloud.Delivery/KenticoCloud.Delivery.csproj (hunk 2 above)
Comment (Simply007) [reply KenticoCloud.Delivery/KenticoCloud.Delivery.csproj]: I have add PublishRepositoryUrl to the csprojs. But the nuspec already had the url un the repository tag.
* 6bdcc37

Comment (Simply007) [reply KenticoCloud.Delivery/KenticoCloud.Delivery.csproj]: Do we want to  leave it here?

File: KenticoCloud.Delivery/KenticoCloud.Delivery.csproj (hunk 1 above)
Comment (petrsvihlik) [reply KenticoCloud.Delivery/KenticoCloud.Delivery.csproj]: ok

File: KenticoCloud.Delivery/KenticoCloud.Delivery.csproj (hunk 2 above)
Comment (petrsvihlik) [reply KenticoCloud.Delivery/KenticoCloud.Delivery.csproj]: I'd leave it as it is now

**Pull request merged**